pymic/offload_stream.py

```diff
@@ -562,6 +562,9 @@ def invoke(self, kernel, *args):
             raise OffloadError("Cannot invoke kernel, "
                                "library not loaded on device")
 
+        # Map None to nullptr
+        args = [arg if arg is not None else 0 for arg in args]
```

Comment:
Lets stick with this for the moment.  If in the future things are changed so scalars are passed as scalars (as opposed to pointers to scalars) and pointers are passed in directly then I think passing a actual nullptr makes sense.